v22.1/backup-and-restore-overview.md

```diff
@@ -20,20 +20,20 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
 
 {% include cockroachcloud/ccloud/backup-types.md %} 
 
-## Backup and restore types
+## Backup and restore product support
 
-The following table outlines the available backup and restore types in CockroachDB. See each of the pages linked in the table for usage examples:
+This table outlines the level of product support for backup and restore features in CockroachDB. See each of the pages linked in the table for usage examples:
+
+Backup / Restore  | Description  | Product Support | CockroachDB Cloud Backup Type Support
+------------------+--------------+-----------------+------------------------
+[Full backup](take-full-and-incremental-backups.html) | An un-replicated copy of your cluster, database, or table's data. A full backup is the base for any further backups. | All products (Enterprise license not required) | <ul><li>Customer-owned backups</li><li>Managed-service backups</li></ul>
```

Comment:
Just a suggestion, for the description - it might flow better for the user, if it is in the form of the user action e.g. 
"Set up a backup schedule to take periodic backups".